cluster/stone-prd-rh01/tenants/rhtap-build-tenant/repository.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: pipelinesascode.tekton.dev/v1alpha1
+kind: Repository
```

Comment:
This `Kind` is not currently supported by tenants-config since it is not allowlisted in app-interface.

@gbenhaim thoughts on permitting it? 

Otherwise we can manually apply it to the cluster since it is a on-off step during the gradual migration.

Reply:
I see. Now that I think about it, I can just drop this file and have Konflux create the Repository automatically when I try to onboard